ACKategories-iOS/ReusableView.swift

```diff
@@ -15,6 +29,17 @@ extension UITableView {
         return dequeueReusableCell(withIdentifier: T.reuseIdentifier, for: indexPath) as! T
     }
 
+    /// Get prototype cell of given type
+    public func prototypeCell<T>(type: T.Type = T.self) -> T where T: UITableViewCell {
+        if let prototype = prototypeCells[T.reuseIdentifier] as? T {
+            return prototype
+        }
+
+        let prototype = T()
```

Comment:
Is it ok to create UITableViewCell like that? 🤔 I would rather use deque or initializer with style and reuse identifier, but actually don't know why 😄

Reply:
I do this for prototype cells all the time, I think that using dequeue is not a good idea as this cell will never appear in tableView.

Reply:
OK, I agree dequeue is not a good idea, anyway initializer with nil reuse identifier seems more legal to me.. But if you say this works well.. 🤷